web-admin/src/features/billing/plans/utils.ts

```diff
@@ -4,14 +4,19 @@ import { DateTime } from "luxon";
 import { writable } from "svelte/store";
 
 export function formatDataSizeQuota(
+  total: number,
```

Comment:
Mind being more specific than "total"? It's hard for me to tell what the unit is